app/modules/intelligence/agents/chat_agents/tool_helpers.py

```
@@ -0,0 +1,178 @@
+            if isinstance(content, List):
+                try:
+                    res = "\n-> retrieved code snippets: \n" + "\n- content:\n".join(
+                        [
+                            f"""
+```{str(node.get('code_content'))[:min(len(str(node.get('code_content'))),600)]+" ..."}
+```
+"""
+                            for node in content
+                        ]
+                    )
+                except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)